### PR TITLE
Refactor duplicate code into set_initial_values method.

### PR DESCRIPTION
**Pull request by eforgacs**

### CODE REVIEW

File: pygameMenu/menu.py [hunk 1]
```diff
@@ -588,6 +580,17 @@ def add_text_input(self,
 
         return widget
 
+    def set_initial_values(self, align, font_size):
```
Comment (anxuae) [inline pygameMenu/menu.py:583]: You means `get_initial_values` ? (this function set nothing)

Comment (anxuae) [reply pygameMenu/menu.py:583]: Sorry for my short comment.
In fact this function is quite ambiguous, it update the widget global position and it process the alignment and font size.

Maybe should be split un 2 private methods

Comment (eforgacs) [reply pygameMenu/menu.py:583]: Good idea. I'll change it and resubmit.

**Comment by ppizarror** [comment]
I hope this PR does not collide with #100, I've done many refactorings in menu class

**Comment by eforgacs** [comment]
It's up to you guys what you want to do, I just saw some duplicate code that I thought could be refactored. No big deal either way

**Comment by ppizarror** [comment]
@eforgacs that's what I mean 😢, can you review this PR please? Thank you 😄

**Comment by eforgacs** [comment]
Hmm. Maybe it would be better to check again now that #100 has been merged. I'll pull the latest and check if it still makes sense to do this refactor.

**Pull request closed without merging**